config/settings.yml

```diff
@@ -1615,7 +1615,10 @@ vanotify:
         form526ez_reminder_email: <%= ENV['vanotify__services__va_gov__template_id__form526ez_reminder_email'] %>
         form5490_confirmation_email: <%= ENV['vanotify__services__va_gov__template_id__form5490_confirmation_email'] %>
         form5495_confirmation_email: <%= ENV['vanotify__services__va_gov__template_id__form5495_confirmation_email'] %>
+        form674_only_confirmation_email: <%= ENV['vanotify__services__va_gov__template_id__form674_only_confirmation_email'] %>
+        form686c_674_confirmation_email: <%= ENV['vanotify__services__va_gov__template_id__form686c_674_confirmation_email'] %>
         form686c_confirmation_email: <%= ENV['vanotify__services__va_gov__template_id__form686c_confirmation_email'] %>
+        form686c_only_confirmation_email: <%= ENV['vanotify__services__va_gov__template_id__form686c_only_confirmation_email'] %>
```

Comment:
I don't see these in AWS Parameter Store. The instructions for how to format them are here: https://depo-platform-documentation.scrollhelp.site/developer-docs/settings-and-parameter-store

Reply:
Thanks! They've been added now.